src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -1,54 +1,55 @@
 <div class="hmcts-header transparent-background">
   <a [routerLink]="'.'" fragment="content" class="govuk-skip-link">Skip to main content</a>
-    <div class="hmcts-header__container">
+  <div class="hmcts-header__container">
 
-        <ng-container [ngSwitch]="logoType">
+    <ng-container [ngSwitch]="logoType">
 
-          <!-- Logo for Judicial Users -->
-          <ng-container *ngSwitchCase="'judicial'">
-              <div style="position: absolute;" *ngIf="showItems">
-                <a href="#">
-                  <img class="img-responsive" alt="Gov UK Logo" src="/assets/images/govuk-crest-jcm.png" width="83px">
-                </a>
-              </div>
-              <div class="govuk-header__logo">
-              </div>
-              <a class="hmcts-header__link" [routerLink]="headerTitle?.url" [attr.aria-label]="headerTitle?.name" >{{headerTitle?.name}}</a>
-          </ng-container>
+      <!-- Logo for Judicial Users -->
+      <ng-container *ngSwitchCase="'judicial'">
+        <div style="position: absolute;" *ngIf="showItems">
```

Comment:
Can we put this inline style into the scss file?

Reply:
Hi @paulhowes-HMCTS - I've moved the inline style into the accompanying SCSS file.